src/super_gradients/examples/predict/detection_predict_with_labels.py

```diff
@@ -0,0 +1,26 @@
+from super_gradients.common.object_names import Models
+from super_gradients.training import models
+from pathlib import Path
+
+# Note that currently only YoloX, PPYoloE and YOLO-NAS are supported.
+from super_gradients.training.datasets import COCODetectionDataset
+
+model = models.get(Models.YOLO_NAS_L, pretrained_weights="coco")
+mini_coco_data_dir = str(Path(__file__).parent.parent.parent.parent.parent / "tests" / "data" / "tinycoco")
+
+dataset = COCODetectionDataset(
+    data_dir=mini_coco_data_dir, subdir="images/val2017", json_file="instances_val2017.json", input_dim=None, transforms=[], cache_annotations=False
+)
+
+# x's are np.ndarrays images of shape (H,W,3)
+# y's are np.ndarrays of shape (num_boxes,x1,y1,x2,y2,class_id)
```

Comment:
This is so confusing...
what are x and y? I thought they are coordinates. 
don't call images x and labels y....

Reply:
4ec8187f3b19bb9da202c961f5ed2401327f7505 changed